acceptance/Gemfile

```diff
@@ -12,10 +12,11 @@ def location_for(place, fake_version = nil)
   end
 end
 
-gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.33')
-gem "beaker-puppet", *location_for(ENV['BEAKER_PUPPET_VERSION'] || "~> 0.13.4")
+gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 4.0')
+gem "beaker-puppet", *location_for(ENV['BEAKER_PUPPET_VERSION'] || "~> 1.0.1")
 gem "beaker-hostgenerator", *location_for(ENV['BEAKER_HOSTGENERATOR_VERSION'] || "~> 1.1")
 gem "beaker-abs", *location_for(ENV['BEAKER_ABS_VERSION'] || "~> 0.4")
+gem "beaker-vmpooler", *location_for(ENV['BEAKER_VMPOOLER_VERSION'])
```

Comment:
I feel like we should pin this to a particular version

Reply:
Done.